x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/SpecBaseIntegrationTestCase.java

```diff
@@ -146,6 +148,25 @@ private Throwable reworkException(Throwable th) {
         URL source = SpecBaseIntegrationTestCase.class.getResource(url);
         Objects.requireNonNull(source, "Cannot find resource " + url);
 
+        return readURLSpec(source, parser);
+    }
+
+    protected static List<Object[]> readScriptSpec(List<URL> urls, Parser parser) throws Exception {
```

Comment:
I think that it doesn't handle edge cases like `/` or `/path/to/`

Reply:
the purpose of this method is to be used inside our testing infrastructure - making it more generic comes at the expense of code complexity and I'd rather have us invest in that when/if the time comes.